Resources/Locale/en-US/_Goobstation/changeling/store/changeling-catalog.ftl

```diff
@@ -1,140 +1,125 @@
-
-# combat
+# Combat
 
 evolutionmenu-combat-armblade-name = Arm Blade
 evolutionmenu-combat-armblade-desc =
-    Reform one of your arms into a grotesque blade, composed of bone and flesh, able to pry open airlocks and cut through your foes like butter.
-    Costs 15 chemicals.
+    We reform one of our arms into a deadly blade.
+    Costs 20 chemicals.
 
 evolutionmenu-combat-boneshard-name = Bone Shard
 evolutionmenu-combat-boneshard-desc =
-    Break off shards of your bone and shape them into a throwing star which embeds into your foes. But a one timer opportinuty.
+    We break off shards of sharp bone to use as projectile weapons.
     Costs 15 chemicals.
 
 evolutionmenu-combat-armor-name = Chitinous Armor
 evolutionmenu-combat-armor-desc =
-    Inflate your body into an all-consuming chitinous mass of armor.
-    Provides extensive protection against physical damage, but less against other types.
-    It massively slows your movement, and maintaining its shape slows chemical generation.
-    WARNING: Requires you to absorb at least 2 organics to use the ability.
+    We turn our skin into tough chitin to protect us from damage. Upkeep of the armor requires a low expenditure of chemicals.
+    WARNING: We must absorb at least 2 organics to use this ability.
     Costs 25 chemicals.
 
 evolutionmenu-combat-shield-name = Organic Shield
 evolutionmenu-combat-shield-desc =
-    Reforms one of your arms into a large, fleshy shield.
-    Blocks attacks automatically, but very brittle.
+    We reform one of our arms into a hard, yet brittle shield.
     WARNING: Requires you to absorb at least 1 organic to use the ability.
     Costs 20 chemicals.
 
 evolutionmenu-combat-shriek-dissonant-name = Dissonant Shriek
 evolutionmenu-combat-shriek-dissonant-desc =
-    You emit an EMP blast, which disables technology in the surrounding area, including radio headsets.
-    Good for escaping cyborgs and security.
+    We shift our vocal cords to release a high-frequency sound that overloads nearby electronics.
     WARNING: Requires you to absorb at least 1 organic to use the ability.
-    Costs 30 chemicals.
+    Costs 20 chemicals.
```

Comment:
its definitely unimportant and also i didnt even realise i changed this to be Inaccurate. whoops

Reply:
resolved but keeping this open since deciding "what is small and what is medium" is a bit vague. i set the cutoff point to be at 35 chemicals for medium, and the only high amount one is transformation sting at 75 chems